jcloud/parsers/get.py

```diff
@@ -0,0 +1,21 @@
+from .helper import _chf
+
+
+def set_get_parser(subparser, resource):
+    get_parser = subparser.add_parser(
+        'get',
+        help=f'Get a {resource.title()}.',
```

Comment:
> Get a job 
 I think this is a bit vague, better to say getting the information of the job or something?